.github/contributing.md

```diff
@@ -0,0 +1,24 @@
+## Submitting issues
+
+### Short version
+
+ * The [**issue template can be found here**](https://raw.github.com/nextcloud/spreed/master/.github/issue_template.md) but be aware of the different repositories! See list below. Please always use the issue template when reporting issues.
+
+### Guidelines
+* Please search the existing issues first, it's likely that your issue was already reported or even fixed.
+  - Go to one of the repositories, click "issues" and type any word in the top search/command bar.
+  - You can also filter by appending e. g. "state:open" to the search string.
+  - More info on [search syntax within github](https://help.github.com/articles/searching-issues)
+* Report the issue using our [template](https://raw.github.com/nextcloud/spreed/master/.github/issue_template.md), it includes all the informations we need to track down the issue.
+
+If your issue appears to be a bug, and hasn't been reported, open a new issue.
+
+Help us to maximize the effort we can spend fixing issues and adding new features, by not reporting duplicate issues.
+
+
+## Contributing to Source Code
+
+Thanks for wanting to contribute source code to the Mail app. That's great!
```

Comment:
Fixed the app name.

For the other thing I guess we should have that in one repo only, so apps can just like to a common source. Doesn't make sense to do this 20 times.